packages/ember-toucan-core/src/-private/components/form/controls/combobox/option.hbs

```diff
@@ -0,0 +1,36 @@
+{{! template-lint-disable require-presentational-children }}
+<li
+  aria-selected={{if @isSelected "true" "false"}}
+  class="my-0 flex cursor-default items-center gap-2 px-2 py-2 leading-4
+    {{this.styles}}
+    {{this.className}}
+    "
+  data-active={{if @isActive "true" "false"}}
+  id="{{@popoverId}}-{{@index}}"
+  role="option"
+  {{on "click" this.onClick}}
+  {{! template-lint-disable no-pointer-down-event-binding }}
+  {{on "mousedown" this.onMousedown}}
+  {{on "mouseover" @onMouseover}}
```

Comment:
Good callout - let me move this! 🙇

Reply:
Good question!  Yeah, I think we probably could.

This gets used in `combobox.hbs` where we yield back `Option`.  We do the following:

```hbs
onMouseover=(fn this.onOptionMouseover index)
```

Is there a way to make this a modifier rather than a component argument when we are yielding back?